File: TestBank/app/src/main/java/monteoliva/testbank/controller/Statement.java

package monteoliva.testbank.controller;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Statement {

    @SerializedName("statementList")
    @Expose
    private List<StatementList> statementList = null;
    @SerializedName("error")
    @Expose
    private Error error;

    public List<StatementList> getStatementList() {
        return statementList;
    }

    public void setStatementList(List<StatementList> statementList) {
        this.statementList = statementList;
    }

    public Error getError() {
        return error;
    }

    public void setError(Error error) {
        this.error = error;
    }

}

File: app/src/main/java/br/com/testeandroidv2/model/statements/gson/Statement.kt
package br.com.testeandroidv2.model.statements.gson

import android.os.Parcelable

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import kotlinx.android.parcel.Parcelize

@Parcelize
data class Statement(@SerializedName("statementList") @Expose var statementList: MutableList<StatementList>? = null,
                     @SerializedName("error")         @Expose var error: Error? = null
) : Parcelable